app/models/location.rb

```diff
@@ -17,6 +17,10 @@ def full_address
     "#{address}, #{postcode}"
   end
 
+  def sorted_ip_addresses
+    ips.sort_by { |ip| ip.address.split('.').map(&:to_i) }
```

Comment:
What's the intention here? Given `ip.address` is a string isn't lexicographical sorting enough?

```ruby
irb(main):001:0> ip1 = '1.1.1.3'
=> "1.1.1.3"
irb(main):002:0> ip2 = '1.1.1.4'
=> "1.1.1.4"
irb(main):003:0> ip3 = '1.1.1.1'
=> "1.1.1.1"
irb(main):004:0> ip4 = '1.1.1.2'
=> "1.1.1.2"
irb(main):005:0> [ip1, ip2, ip3, ip4].sort
=> ["1.1.1.1", "1.1.1.2", "1.1.1.3", "1.1.1.4"]
```

I'm not sure this method is needed.

Reply:
Resolving as discussed (lexicographical sorting doesn’t account for `.` characters)

Reply:
Great - yes my example is a bit simplisitic ordering when ip begins with `100.` vs. `1.` would be wrong :+1: